src/views/activity/Activity.vue

```diff
@@ -41,12 +41,37 @@ div
       router-link.nav-link(:to="{ name: 'activity-view', params: {...$route.params, view_id: view.id}}" :class="{'router-link-exact-active': currentView.id == view.id}")
         h6 {{view.name}}
 
-    li.nav-item(style="margin-left: auto")
-      a.nav-link(@click="$refs.new_view.show()")
-        h6
-          icon(name="plus")
-          span.d-none.d-md-inline
-            | {{ $t('newView') }}
+    div.edit(v-if="editing" style="margin: auto 1rem 1rem auto;").mt-2
+      div.d-flex.flex-row-reverse
+        b-button(variant="outline-dark" @click="discard(); editing = !editing;")
+          icon(name="times")
+          span Cancel
```

Comment:
All strings need to be in translation tag like this:
```suggestion
          span {{ $t('Cancel') }}
```

And then strings need to be added to locale file.